src/coreclr/unwinder/arm64/unwinder.cpp

```diff
@@ -33,12 +33,13 @@
 #define FIELD_OFFSET(type, field)    ((LONG)__builtin_offsetof(type, field))
 #endif
 
-#ifdef HOST_UNIX
-#define RtlZeroMemory ZeroMemory
-
 #if defined(TARGET_ARM64)
```

Comment:
@kunalspathak should this be `if !defined(DACCESS_COMPILE) && defined(TARGET_ARM64)` ? The same on [Line:273](https://github.com/dotnet/runtime/pull/110472/commits/a99f6391e0a6ea4ce6036e8aae87a20b18d2c195#diff-8d8eea22f227bfd092bc709d426a8e62e6d68ec03c03fcd5b36b579bb78ba729R273)

Reply:
The only place where `PacStripPtr` is getting used in this file is in ` !defined(DACCESS_COMPILE)`, so doesn't harm to keep it that way...so you might need `#if defined(TARGET_ARM64) && !defined(DACCESS_COMPILE)` here.